estate/models/estate_property.py

```diff
@@ -0,0 +1,106 @@
+from odoo import api, fields, models, exceptions, tools
+
+
+class Property(models.Model):
+    _name = "estate.property"
+    _description = "Real Estate property"
+    _order = "sequence, id desc"
+
+    sequence = fields.Integer("Sequence", default=1)
+    name = fields.Char("Title", required=True)
+    description = fields.Text("Description")
+    postcode = fields.Char("Postcode")
+    date_availability = fields.Date(
+        "Available Date", copy=False, default=fields.Date.add(fields.Date.today(), months=3))
+    expected_price = fields.Float("Expected price", required=True)
+    selling_price = fields.Float("Selling price", readonly=True, copy=False)
+    bedrooms = fields.Integer("Bedrooms", default=2)
+    living_area = fields.Integer("Living area (sqm)")
+    facades = fields.Integer("Facades")
```

Comment:
usually when a field is basically a counter, you postfix with '_count'
because 'facade' only is not specific enough